cli_reference/basic_cli_operations.adoc

```diff
@@ -368,6 +368,21 @@ Stop a build that is in progress:
 $ oc cancel-build <build_name>
 ----
 
+Cancel multiple builds at the same time:
+----
+$ oc cancel-build <build1_name> <build2_name> <build3_name>
+----
+
+Cancel all builds created from the build configuration:
+----
+$ oc cancel-build bc/foo
```

Comment:
Maybe instead of `foo` use `<buildconfig_name>`. Same for the other instances of `foo` throughout.